packages/devextreme-react/src/core/__tests__/template.test.tsx

```diff
@@ -301,68 +391,94 @@ function testTemplateOption(testedOption: string) {
       )),
     };
 
-    const component = mount(React.createElement(ComponentWithTemplates, elementOptions));
-    const componentInstance = component.instance() as any;
+    const { container } = render(
+    <ComponentWithTemplates {...elementOptions} ref={ref}>
+      <div ref={refContainer} />
+    </ComponentWithTemplates>
+    );
+
+    const componentInstance = ref.current as unknown as { _templatesStore: TemplatesStore};
+
+    renderItemTemplate(undefined, refContainer.current);
 
-    renderItemTemplate();
     expect(componentInstance._templatesStore.renderWrappers().length).toBe(1);
-    component.update();
-    const templateContent = component.find('.template').getDOMNode();
+
+    const templateContent = container.querySelector('.template') as HTMLElement;
 
     const { parentElement } = templateContent;
     if (!parentElement) { throw new Error(); }
 
     parentElement.removeChild(templateContent);
     events.triggerHandler(templateContent, 'dxremove');
-    component.update();
+
     expect(componentInstance._templatesStore.renderWrappers().length).toBe(0);
   });
 }
 
 describe('function template', () => {
+  jest.useFakeTimers();
```

Comment:
We call `jest.useFakeTimers()` in this `describe`, but don't in next `describe`. Should we call it globally?